packages/synapse-react-client/src/components/file/upload/UploadFilePanel.tsx

```diff
@@ -41,6 +41,11 @@ export type UploadFilePanelProps = {
    * @default false
    */
   disabled?: boolean
+  /**
+   * Whether to enable drag-and-drop to upload functionality.
+   * @default false
+   */
+  disableDragAndDrop?: boolean
```

Comment:
The drag & drop handler requires manual set up (today, done only in SWC), so this prop can be used to hide UI that communicates whether drag & drop is available.